src/structures/GroupChat.js

```diff
@@ -195,8 +196,11 @@ class GroupChat extends Chat {
         return await this.client.pupPage.evaluate(async (chatId, participantIds) => {
             const chat = await window.WWebJS.getChat(chatId, { getAsModel: false });
             const participants = participantIds.map(p => {
-                return chat.groupMetadata.participants.get(p);
-            }).filter(p => Boolean(p));
+                const wid = window.Store.WidFactory.createWid(p);
+                const lid = wid.server!=='lid' ? window.Store.LidUtils.getCurrentLid(wid) : wid;
+                const phone = wid.server=='lid' ? window.Store.LidUtils.getPhoneNumber(wid) : wid;
+                return (chat.groupMetadata.participants.get(lid._serialized) || chat.groupMetadata.participants.get(phone._serialized));
+            });
```

Comment:
We can simply here for more readability:

```suggestion
                const id = window.Store.WidFactory.createWid(p);
                const lidOrWid = id.server === 'lid'
                    ? window.Store.LidUtils.getPhoneNumber(id)
                    : window.Store.LidUtils.getCurrentLid(id);
                    
                return chat.groupMetadata.participants.get(lidOrWid?._serialized);
            }).filter(p => Boolean(p));
```

Also the resulted array have to be filtered out of all the undefined objects that can be there.